crates/sui-graphql-rpc/schema/draft_target_schema.graphql

```diff
@@ -780,7 +780,7 @@ type Checkpoint {
 }
 
 type TransactionBlock {
-  digest: String!
+  digest: String
```

Comment:
I'm sure I'll learn why in a bit -- but reading this now, I had the question of why this is optional now (a digest is a key for a transaction block, so we should always be able to provide it, right?)

Reply:
Answering my own question (let me know if I'm wrong) it's the `DryRun` case that's the issue here -- the digest is the digest of the `SenderSignedData`, so we can't compute one for a dry run transaction, because it isn't sender signed.